### PR TITLE
Expand user admin additional filters if form data has any values

### PR DESCRIPTION
**Pull request by kipanshi**
- Improves user experience, if any filter has values entered, the filter group will be expanded

### CODE REVIEW

**Comment by kipanshi** [comment]
Tested locally, can provide gif

File: src/Presenters/UsersAdminPresenter.php [hunk 1]
```diff
@@ -314,6 +314,12 @@ public function createComponentAdminFilterForm()
         $form->onSuccess[] = [$this, 'adminFilterSubmitted'];
 
         $form->setDefaults($this->adminFilterFormData->getFormValues());
+
+        // Expand extra filters group if any of the search params are present
```
Comment (kipanshi) [inline src/Presenters/UsersAdminPresenter.php]: Another option is to move the logic to line 269 where the div class is set originally

Comment (rootpd) [reply src/Presenters/UsersAdminPresenter.php]: Thanks for the PR! :slightly_smiling_face: 

The end of the factory seems like the better place, dataproviders could affect the configured defaults. We amended it a bit so it only expands if the collapsed group have some value to filter. Once it's merged in the master and pushed to this repository, we'll make a comment here.

Comment (kipanshi) [reply src/Presenters/UsersAdminPresenter.php]: Great, thank you, looking forward to the release, this will improve the user experience for our customers.

**Pull request closed without merging**